code-challenges.test.js

```diff
@@ -26,17 +24,52 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
+//Write function named divideThree with the parameter/argument named num
 
+const divideThree = (num) => {
 
 
 
-// b) Create the function that makes the test pass.
+    
+//Write if statment if num is divided by three it is strictly equal to zero.
+    if(num % 3 === 0){
 
+//return a string interpolation stating num is divisible by three
+        return `${num} is divisible by three`
+
+//write an else if statement of num divided by three is not equal to zero
+    } else if(num % 3 !==0){
+
+// return with string interpolation stating num is not divisible by three
+        return `${num} is not divisible by three`
+// input else statement to output something is wrong if there is an error.
+    } else {
+        return "Something is wrong"
+    }
+}
+
+// a) Create a test with expect statements for each of the variables provided.
+
+//write test with the method "divideThree" to return is divisible by three.
+describe("divideThree", () => {
+    test ("returns is divisible by three", () => {
+        expect(divideThree().toEqual("is divisible by three"))
+    })
+})
+
+
+// b) Create the function that makes the test pass.
+//write the function diviethree to return is divisible by three.
+divideThree = () => {
```

Comment:
By making another function and/or method of the same name we will throw off our computer pretty hard. Computers require us to be specific and consistent. If we give it two functions of the same name it will be confused.